website/js/components/Question.js

```diff
@@ -19,6 +19,7 @@ export default ({
         tags.includes(state.filter.category.toLowerCase())
     })}
     key={name}
+    id={name.substring(0, name.length - 3)}
```

Comment:
I prefer `slice()` but it's no big deal

Reply:
Could you explain this one? I kinda feel lost using `.slice()` to cut last 3 characters. 😢